std/sync/svs_alo.go

```diff
@@ -2,6 +2,7 @@ package sync
 
 import (
```

Comment:
I'm confused in this class:
- There is no storage.
- I do not find a feedback from the client side.

Then, how do you ensure the object is delivered at least once after disaster recovery (i.e. process killed)?

Reply:
The callback returning is the feedback. If the application needs to block, it should block the callback.